docs/sentiment-analysis.md

```diff
@@ -0,0 +1,8 @@
+Sentiment analysis is the process of using text analysis to
+identify and study states of emotion in regards to the input, which is subjective information.
+
+Sentiment analysis is able to identify the amount of sentiment each user has
+for what it is processing. It will give numbers based on each users sentiment.
```

Comment:
I think these two sections here have very similar meaning and should be combined